tests/test_volume.py

```diff
@@ -160,6 +161,14 @@ def testProject(self):
             #  centered along the rotation axis for multiples of pi/2.
             self.assertTrue(np.allclose(vol_along_axis, prj_along_axis))
 
+        # test with saved ribosome data
+        results = np.load(os.path.join(DATA_DIR, "clean70SRibosome_down8_imgs32.npy"))
+        vols = Volume(np.load(os.path.join(DATA_DIR, "clean70SRibosome_vol_down8.npy")))
+        rots = np.load(os.path.join(DATA_DIR, "rand_rot_matrices32.npy"))
+        rots = np.moveaxis(rots, 2, 0)
+        imgs_clean = vols.project(0, rots).asnumpy()
+        self.assertTrue(np.allclose(results, imgs_clean, atol=1e-7))
+
```

Comment:
Nice. I guess this is essentially a hardcoded test since `"clean70SRibosome_down8_imgs32.npy"` were produced with `project` to begin with, iirc.